<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssociateElementWin_ui</class>
 <widget class="QDialog" name="AssociateElementWin_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Associa Elementi</string>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout">
       <item row="0" column="0">
        <widget class="QLineEdit" name="leftEditor"/>
       </item>
       <item row="1" column="0">
        <widget class="QListWidget" name="leftList"/>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>97</width>
        <height>192</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>97</width>
        <height>1600000</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="btn_arrow">
         <property name="text">
          <string>--&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="btn_add">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="btn_line">
         <property name="text">
          <string>--</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout">
       <item row="0" column="0">
        <widget class="QLineEdit" name="rightEditor"/>
       </item>
       <item row="1" column="0">
        <widget class="QListWidget" name="rightList"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leftEditor</tabstop>
  <tabstop>leftList</tabstop>
  <tabstop>rightEditor</tabstop>
  <tabstop>rightList</tabstop>
  <tabstop>btn_arrow</tabstop>
  <tabstop>btn_line</tabstop>
  <tabstop>btn_add</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
